index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/5e67ba493b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="socials">
+          <ul>
+            <li>
+              <a href="https://www.facebook.com/"
+                ><img src="./assets/facebook.png" alt="facebook logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/nerdy_me_"
+                ><img src="./assets/twitter.png" alt="twitter logo"
+              /></a>
+            </li>
+            <li><a href="">English</a></li>
+            <li><a href="">MY Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
```

Comment:
- [ ] Kindly adjust the size of your social icons to match the original design as they appear to be bigger. Also, adjust the color of the Twitter and facebook icons.
 
![image](https://user-images.githubusercontent.com/98453979/217300512-f89b5a06-b0d8-4471-8cb9-bc274fdf7e07.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/217300999-34b3e513-6f00-4fe0-8a12-853c13c1a26b.png)